shared/chat/conversation/info-panel/participant.js

```diff
@@ -29,11 +23,7 @@ const Participant = ({username, following, fullname, broken, isYou, onShowProfil
         >
           <Avatar size={isMobile ? 40 : 32} username={username} />
           <Box style={{...globalStyles.flexBoxColumn, marginLeft: globalMargins.small}}>
-            <Usernames
```

Comment:
try and use connected usersnames where possible to reduce the props we need to pass